lib/Drawers/Drawers.js

```diff
@@ -235,39 +242,69 @@ module.exports = kind(
 	* @private
 	*/
 	setupHandles: function () {
-		var controls, index,
-			handles = [];
+		var handle,
+			def = {kind: Item, owner: this},
+			tap = this.bindSafely(this.handleTapped);
+
+		this.drawers.forEach(function (drawer) {
```

Comment:
Similar question to before - think we should switch to a standard `for` loop here?